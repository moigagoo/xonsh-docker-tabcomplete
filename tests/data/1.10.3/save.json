{
        "arguments": [
                "--help",
                "-o", "--output"
        ],
        "commands": []
}
