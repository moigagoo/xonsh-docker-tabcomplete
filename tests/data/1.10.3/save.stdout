
Usage:	docker save [OPTIONS] IMAGE [IMAGE...]

Save an image(s) to a tar archive (streamed to STDOUT by default)

  --help             Print usage
  -o, --output       Write to a file, instead of STDOUT
